app/views/ReportView/components/Slides/components/UploadSlide/index.tsx

```diff
@@ -66,7 +66,7 @@ const UploadSlide = ({
   }, [onUpload, slideName, snackbar, report]);
 
   return (
-    <>
+    <div className="upload">
       <TextField
         className="text-field-fix"
```

Comment:
seems like the `.text-field-fix` causes margin problems 
![image](https://user-images.githubusercontent.com/25511396/115922852-46ea1c80-a432-11eb-91c8-56d4971647a3.png)